README.md

```diff
@@ -74,11 +82,30 @@ For actually tracing, Tracee needs to run with sufficient capabilities:
 
 Alternatively, running as `root` or with the `--privileged` flag of Docker, is an easy way to start.
 
+#### Linux Headers
+
+In order to compile the eBPF program, Tracee needs some of the Linux kernel headers. Depending on your Linux distribution, there are different ways to obtain it, and different places they end up at.  
+
+> Note that it's important that the kernel headers match the exact version of kernel you are running. To check the current kernel version, run the command `uname -r`
+
+Here are a few examples of how to obtain Linux kernel headers in different Linux distributions:
```

Comment:
You write a few, but give only one example

Reply:
yes, I meant to ask you about manjaro and simar about centos

Reply:
so how do you do it on manjaro? @yanivagman